src/modules/layout/containers/LocationLayout.js

```diff
@@ -13,19 +11,22 @@ import GeneralFooter from '../components/GeneralFooter'
 import LocationHeader from '../components/LocationHeader'
 import LocationFooter from '../components/LocationFooter'
 
-export class LocationLayout extends React.Component {
-  static propTypes = {
-    matchRoute: PropTypes.func.isRequired,
-    location: PropTypes.string.isRequired,
-    language: PropTypes.string.isRequired,
-    locations: PropTypes.arrayOf(PropTypes.instanceOf(LocationModel)).isRequired,
-    currentPath: PropTypes.string.isRequired,
-    viewportSmall: PropTypes.bool.isRequired,
-    children: PropTypes.node,
-    events: PropTypes.arrayOf(PropTypes.instanceOf(EventModel)).isRequired
-  }
+import type { EventType } from '../../endpoint/types'
+import Route from '../../app/Route'
+
+type Props = {
```

Comment:
As we no longer use prop-types, does it check in runtime that all props are passed? Or is this Props thing a compile-time feature?